### ISSUE TITLE
[@brillout/vite-plugin-import-build@0.2.10][DEBUG] requireError undefined ?

### ISSUE THREAD

**Issue by brillout** [brillout/vite-plugin-ssr#672]
### Discussed in https://github.com/brillout/vite-plugin-ssr/discussions/671

<div type='discussions-op-text'>

<sup>Originally posted by **fortezhuo** February 25, 2023</sup>
When I run my app that built using vite-plugin-ssr. Logs print out

```
[@brillout/vite-plugin-import-build@0.2.10][DEBUG] requireError undefined
[@brillout/vite-plugin-import-build@0.2.10][DEBUG] success true
[@brillout/vite-plugin-import-build@0.2.10][DEBUG] DEBUG_LOGS_RUNTIME [end]
```

What's requireError undefined mean ? and how to tweak or disable debug log ?</div>

**Comment by brillout** [comment]
@fortezhuo It's a bug, let me fix it.

**Comment by brillout** [comment]
Fix released in `0.4.89`.
